src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/ReportRepository.kt

```diff
@@ -115,7 +115,7 @@ class OrderlyReportRepository(val isReviewer: Boolean,
                     .fetch()
 
             return versions.groupBy { r -> r["name"] }.map {
-                it.value.maxBy { r -> r["latestVersion"] as String }
+                it.value.maxByOrNull { r -> r["latestVersion"] as String }
```

Comment:
`maxByOrNull` maybe throwing the `UnsafeCallOnNullableType`, to fix this, you may want to add additional test for testing nullable.

Reply:
Yeah, have silenced the linter to stop this from alerting. In fact this should never be null and this code should be re-written to avoid using `!!` here at all (but not in scope for this ticket)